Services/Dashboard/ItemsBlock/classes/class.ilPDSelectedItemsBlockViewGUI.php

```diff
@@ -1,17 +1,17 @@
 <?php
```

Comment:
Thx for the refactoring. I would highly appreciate if you could also add strcit type here.

Reply:
Removed strict type because the code is not type strict.